src/bin/pg_autoctl/fsm_transition.c

```diff
@@ -353,7 +355,7 @@ fsm_disable_replication(Keeper *keeper)
  *
  *    start_postgres
  * && disable_synchronous_replication
- * && drop_replication_slot
+ * && keeper_drop_replication_slots_for_removed_nodes
```

Comment:
The name of the action here does not have to match the name of the function in the source code, but that's fine too.